assets/js/util.js

```diff
@@ -308,17 +332,21 @@ function callApy(options, endpoint) {
     var requestUrl = window.location.protocol + window.location.hostname +
         window.location.pathname + '?' + $.param(requestOptions.data);
 
+    if(useAjax) {
+        return $.ajax(requestOptions);
```

Comment:
> Doing this will help further where we would be able to wrap this around $.jsonp().

I foresee this not actually ever happening :)

> Do you suggest putting this useAjax flag along with the check condition something like:

That wouldn't work since then we'll always be using POST if `useAjax` is set rather than switching between GET and POST, right?

> Does the following approach make sense?

It works. Is there a less repetitive option though?